### ISSUE TITLE
Discovery: evaluate timoni.io

### ISSUE THREAD

**Issue by piontec** [giantswarm/roadmap#2356]
- [ ] https://github.com/giantswarm/roadmap/issues/2356#issue-1664301708 needs to be done first

**Comment by piontec** [comment]
From kubecon amsterdam

- Stefan's point of view: Helm is dying…
        - Most of the devs already left
            - A guy that wrote CRD handling left long time ago and nobody wants to touch this code
        - There will no new features in Helm, maintenance mode only, only patches
            - And even this sucks: it took Flux guys 6 months to get a security PR around custom CA handling merged
        - Not possible to get out of the YAML craziness
            - piontec: I fully agree
        - CUE has types, modules, full validation resulting from types you already have
        - That's why he started Timoni
    - Timoni is a private project
        - Contributions welcome, he does it in his own free time, Weave is not currently interested
        - He met the CUE guys, happy about cooperation and how helpful they are
            - Although Kubernetes is just 1 more use case for them, nothing special
        - Helm is dangerous, some day might be just abandoned, yet almost everyone is using it
        - Let's try to convert people
        - Importing helm charts is hard, as they are just a bunch of text glued together. But maybe some form of import into structured types of cue will be possible
        - If all goes right, there's gonna be a timoni-controller in flux

**Comment by piontec** [comment]
We should at least give it a try.

**Comment by weatherhog** [comment]
@piontec this one is old, is it still valid or can it be closed?

**Comment by piontec** [comment]
Yes, it is and it's still the most interesting helm alternative I've seen, so probably worth a look. And it's from Stefan Prodan, which is a recommendation on it's own.

**Comment by weatherhog** [comment]
timoni.io seems to be dead. Closing this issue

**Comment by piontec** [comment]
Very much alive, just moved the page :D https://timoni.sh/
